test/IRGen/arm_to_thumb_darwin.sil

```diff
@@ -0,0 +1,28 @@
+// RUN: %swift -target armv7-apple-ios7 %s -gnone -emit-ir -o - | %FileCheck %s -check-prefix=IOS
+// RUN: %swift -target armv7k-apple-watchos2 %s -gnone -emit-ir -o - | %FileCheck %s -check-prefix=WATCHOS
+
+// REQUIRES: CODEGENERATOR=ARM
```

Comment:
This works (builtin_word.sil also does this and I tested locally with an x86_64 build only). There is no import of the standard library.